.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules
 .DS_Store
 yarn.lock
 package-lock.json
+package.json
```

Comment:
Listing package.json and package-lock.json in here isn't the best idea😅 There're might be some situations when you do want and need to commit them

Reply:
nooo, Didn't you tell me not to add the package.json ..i'll check

Reply:
I guess it was me😅 But people get smarter and opinions change.. so let’s say I was just a little bit too dumb))

But, actually, I’ve always thought that we should commit these two files when we’ve installed new packages. This way whoever fork/clone this repo will install all the  necessary packages (responsible for that is `package.json`) with the same versions (  responsible for that is `package-lock.json`) (Commands to install `npm install/yarn install)`

Reply:
And if I’m being honest, it was Illya who told us to not do that… So, I’m not the root of this problem😆

Reply:
Not only you, Dima, many people told me not to commit the file. I'm lost😁